demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1270,6 +1283,63 @@ static int32_t connectToS3Server( NetworkContext_t * pNetworkContext,
 
 /*-----------------------------------------------------------*/
 
+static OtaHttpStatus_t handleHttpResponse( const HTTPResponse_t * pResponse )
+{
+    /* Return error code. */
+    OtaHttpStatus_t ret = OtaHttpRequestFailed;
+
+    OtaEventData_t * pData;
+    OtaEventMsg_t eventMsg = { 0 };
+
+    switch( pResponse->statusCode )
+    {
+        case HTTP_RESPONSE_PARTIAL_CONTENT:
+            /* Get buffer to send event & data. */
+            pData = otaEventBufferGet();
+
+            if( pData != NULL )
+            {
+                /* Get the data from response buffer. */
+                memcpy( pData->data, pResponse->pBody, pResponse->bodyLen );
+                pData->dataLength = pResponse->bodyLen;
+
+                /* Send job document received event. */
+                eventMsg.eventId = OtaAgentEventReceivedFileBlock;
+                eventMsg.pEventData = pData;
+                OTA_SignalEvent( &eventMsg );
+
+                ret = OtaHttpSuccess;
+            }
+            else
+            {
+                LogError( ( "Error: No OTA data buffers available.\r\n" ) );
```

Comment:
\r\n are automatically added to the print statements so they aren't needed here. Unless you are intending to create extra spacing, in that case it's okay.

Reply:
Addressed in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1451/commits/463ab1e0eb8ae36da14dfb76e6ab62d316ff0ab1